packages/sanity/src/_internal/cli/commands/dataset/importDatasetCommand.ts

```diff
@@ -105,8 +105,15 @@ const importDatasetCommand: CliCommandDefinition = {
   action: async (args, context) => {
     const {apiClient, output, chalk, fromInitCommand} = context
     const flags = parseFlags(args.extOptions)
-    const {allowAssetsInDifferentDataset, allowFailingAssets, assetConcurrency, replaceAssets} =
-      flags
+    const {
+      allowAssetsInDifferentDataset,
+      allowFailingAssets,
+      assetConcurrency,
+      skipCrossDatasetReferences,
+      replaceAssets,
+      replace,
+      missing,
```

Comment:
Same as below, these flags aren't actually used. Not sure why we're parsing them here. Maybe the `getMutationOperation` logic should be moved in here instead?

Reply:
I see yes. I'm happy to just remove them from here, I don't see any big benefit to inlining `getMutationOperation`